packages/block-library/src/file/index.php

```diff
@@ -19,6 +19,28 @@ function render_block_core_file( $attributes, $content ) {
 		wp_enqueue_script( 'wp-block-file-view' );
 	}
 
+	// Update object's aria-label attribute if present in block HTML.
+
+	// Match an aria-label attribute from an object tag.
+	$pattern = '@<object.+(?<attribute>aria-label="(?<filename>[^"]+)?")@i';
+	$content = preg_replace_callback(
+		$pattern,
+		function ( $matches ) {
+			$filename     = ! empty( $matches['filename'] ) ? $matches['filename'] : '';
```

Comment:
I originaly though of doing that but I discovered that `fileName` was not present in the `$attributes` array in PHP since its sourced from the HTML of the block.